web/middleware/auth.ts

```diff
@@ -29,23 +36,22 @@ export function authCheckFactory(optional = false, useQueryParams = false, loadE
                 where: [
                     {
                         authToken: hashToken(token),
```

Comment:
How can we use a dropped field here?

Reply:
I think we can remove the auth check factory entirely, as all remaining REST Apis should be unauthenticated / use the GraphQL Session

Reply:
Good Catch!

Reply:
> How can we use a dropped field here?

Typescript is never as safe as it promises 😞 
The where attibute includes the `ObjectLiteral` which basically accapts any object
```
    where?: FindConditions<Entity>[] | FindConditions<Entity> | ObjectLiteral | string;

    export interface ObjectLiteral {
      [key: string]: any;
    }
```

> as all remaining REST Apis should be unauthenticated / use the GraphQL Session

What you mean is that for example the user API is getting/validating the user by it's session already? So I could just remove the `authCheckFactory` middleware everywhere?

Reply:
No, we could just remove the user API.

Reply:
Then I would propose to keep it as it is for now and remove the old apis in different commits. 
The broken `authCheckFactory` will make sure that no user data will be leaked until then.

Reply:
Mm can we explicitly disable the authCheckFactory? Not a fan of broken authentication

Reply:
Done